tutorials/clinical/10_ieeg_localize.py

```diff
@@ -199,24 +200,38 @@ def plot_overlay(image, compare, title, thresh=None):
 #
 #     $ freeview $MISC_PATH/seeg/sample_seeg_T1.mgz \
 #           $MISC_PATH/seeg/sample_seeg_CT.mgz
+
+###############################################################################
+# Now, we'll make a montage with the channels that we've found in the
+# previous step.
 #
-# Now, we'll need the subject's brain segmented out from the rest of the T1
-# image from the freesurfer ``recon-all`` reconstruction. This is so that
-# we don't have extraneous data outside the brain affecting our warp to a
-# template brain.
-#
-# Let's plot the electrode contact locations on the subject's brain.
+# .. note:: MNE represents data in the "head" space internally
 
 # load electrode positions from file
 elec_df = pd.read_csv(op.join(misc_path, 'seeg', 'sample_seeg_electrodes.tsv'),
                       sep='\t', header=0, index_col=None)
 ch_names = elec_df['name'].tolist()
 ch_coords = elec_df[['R', 'A', 'S']].to_numpy(dtype=float)
 
+# get fiducial points for our subject from the Talairach transform
+# the was computed during the Freesurfer recon-all
+lpa, nasion, rpa = mne.coreg.get_mni_fiducials(
+    'sample_seeg', subjects_dir=op.join(misc_path, 'seeg'))
+lpa, nasion, rpa = lpa['r'], nasion['r'], rpa['r']
+
+# create a montage in subject-specific brain space
+ch_pos = dict(zip(ch_names, ch_coords / 1000))  # mm -> m
+montage = mne.channels.make_dig_montage(
+    ch_pos, coord_frame='mri', nasion=nasion, lpa=lpa, rpa=rpa)
+
+###############################################################################
+# Let's plot the electrode contact locations on the subject's brain.
+
 # load the subject's brain
-subject_brain = nib.load(op.join(misc_path, 'seeg', 'sample_seeg_brain.mgz'))
+subject_brain = nib.load(
+    op.join(misc_path, 'seeg', 'sample_seeg', 'mri', 'brain.mgz'))
 
-# make brain surface from T1
+# Make brain surface from T1
 verts, triangles = mne.surface.marching_cubes(
```

Comment:
Actually now that we have proper `pial` surfaces in the dataset, can you just load and plot them instead? It should further simplify the example because you don't have to marching cubes + transform, just load and plot

Reply:
(and actually you can use `mne.viz.plot_alignment` to do this for you in one line!)

Reply:
Sure, sounds good

Reply:
Actually can you do this in one of the other PRs? It seems like getting this PR merged is a bottleneck for the others, so I'd rather wait for CIs now and merge this ASAP

Reply:
So we took out the `bem` surfaces in the paring down so we can't plot the head...

Maybe a PR for a later date

Reply:
I don't think it's critical to show the head but if we need to update the dataset at some point we could add it